### ISSUE TITLE
Database Backup

### ISSUE THREAD

**Issue by snarfblam** [snarfblam/Prop-manager#67]
This is an important feature. At the very least, an administrator should be able to download the database as a json file. Even if we don't initially provide a method of restoring the database from a backup, it gives the admin a way to avoid total data loss in the event of a hardware failure or catastrophic bug.

Down the road, we would want to have some kind of automated backup and a means to restore from backups.

**Comment by aknight023** [comment]
I believe JAWs db has automated backups. i remember seeing that somewhere.

**Comment by snarfblam** [comment]
You're right. [JawsDB plans and pricing](https://elements.heroku.com/addons/jawsdb)

It may still be nice to be able to dump/insert the database to switch between services or if you want to use a service that doesn't offer automatic backups (though I'm guessing anything but self-hosting comes with backups). Even then, it's possible to do these things via command line or MySQL Workbench.
